js/glpi_dialog.js

```diff
@@ -304,3 +304,39 @@ var glpi_confirm = function({
 var glpi_close_all_dialogs = function() {
    $('.modal.show').modal('hide').remove();
 };
+
+var toast_id = 0;
+var glpi_toast = function(title, message, css_class) {
+   toast_id++;
+
+   var html = `<div class='toast-container bottom-0 end-0 p-3 messages_after_redirect'>
+      <div id='toast_js_${toast_id}' class='toast ${css_class} animate__animated animate__tada animate__delay-2s animate__slow' role='alert' aria-live='assertive' aria-atomic='true'>
+         <div class='toast-header'>
+            <strong class='me-auto'>${title}</strong>
+            <button type='button' class='btn-close' data-bs-dismiss='toast' aria-label='${__('Close')}'></button>
+         </div>
+         <div class='toast-body'>
+            ${message}
+         </div>
+      </div>
+   </div>`;
+   $('body').append(html);
+
+   var toast = new bootstrap.Toast(document.querySelector('#toast_js_' + toast_id), {
+      delay: 10000,
+   });
+   toast.show()
+}
+
+var glpi_toast_info = function(message) {
+   glpi_toast(_n("Informations", "Information", 1), message, 'bg-info text-white border-0');
```

Comment:
```suggestion
   glpi_toast(_n("Information", "Information", 1), message, 'bg-info text-white border-0');
```

Reply:
Done + fixed the same locale in two others places -> 990bf707f4586617397b4fed476fee59bffcfafd.